{
  "Header": {
    "Original": "CVE-2025-38713 (In the Linux kernel, the following vulnerability has been resolved:  h ...)",
    "ID": "CVE-2025-38713",
    "Description": "(In the Linux kernel, the following vulnerability has been resolved:  h ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-6009-1 DLA-4328-1 DLA-4327-1}",
      "Type": "xref",
      "Bugs": [
        "DSA-6009-1",
        "DLA-4328-1",
        "DLA-4327-1"
      ]
    },
    {
      "Original": "- linux 6.16.3-1",
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.16.3-1"
    },
    {
      "Original": "[trixie] - linux 6.12.43-1",
      "Type": "package",
      "Release": "trixie",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.12.43-1"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/94458781aee6045bd3d0ad4b80b02886b9e2219b (6.17-rc1)",
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/94458781aee6045bd3d0ad4b80b02886b9e2219b (6.17-rc1)"
    }
  ]
}